src/rf/js/src/core/models.js

```diff
@@ -61,15 +79,15 @@ var Layer = Backbone.Model.extend({
     },
 
     isPreValidated: function() {
-        return this.get('status') === STATUS_CREATED;
+        return this.get('status_created');
```

Comment:
There are other points in the app that reference isPreValidated such as existsUploading in PendingLayers. I think that stuff needs to be reworked to directly use the uploading / copying statuses.